core/src/main/scala/kafka/server/ConfigHandler.scala

```diff
@@ -62,6 +62,15 @@ class TopicConfigHandler(private val logManager: LogManager, kafkaConfig: KafkaC
     topicConfig.asScala.forKeyValue { (key, value) =>
       if (!configNamesToExclude.contains(key)) props.put(key, value)
     }
+
+    val remoteLogStorageEnable = topicConfig.getProperty(TopicConfig.REMOTE_LOG_STORAGE_ENABLE_CONFIG)
+
+    if (!kafkaConfig.remoteLogManagerConfig.enableRemoteStorageSystem()
```

Comment:
Instead of 

`kafkaConfig.remoteLogManagerConfig.enableRemoteStorageSystem()` ->

`kafkaConfig.isRemoteLogStorageSystemEnabled`